### ISSUE TITLE
Using partials corrupts the `xlsx` file.

### ISSUE THREAD

**Issue by henrikbjorn** [caxlsx/caxlsx_rails#186]
Having a controller template doing

``` rb
render 'statistics/shared/workbook`
```

and the `_workbook` partial being something like:

``` rb

book = xlsx_package.add_worksheet
sheet = book.add_worksheet(name: "something")

sheet.add_row ['row', 'boat', 'show']
```

Makes the generated xlsx file corrupt. Moving the partial contents to the template directly and the file is not corrupt and work as it should.

**Comment by straydogstudio** [comment]
@henrikbjorn  Are you directly rendering a partial without a regular template?

**Comment by henrikbjorn** [comment]
Nope, i have a regular template which is rendering a partail. To share the xlsx generation I have a `index` template, a `show` template etc.

**Comment by straydogstudio** [comment]
Partials are used regularly in our tests without issue. Can you share gem versions for rails, caxlsx, caxlsx_rails? Can you duplicate it in a bare bones rails application?
